### ISSUE TITLE
Progress bar updates are not announced to screen reader users

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3516]
## Describe the bug

If a progress bar indicates updates that are meaningful, they are not announced.

## How to reproduce

https://clarity.design/documentation/progress

- Using a screen reader, activate the first 'Show' button on the page and observe that you do not hear any information about the progress bar updates that are occurring visually

## Expected behavior

Use a live region to announce progress updates as seen at
http://chris.homepages.org/clane/portfolio/ariaProgressBar/ariaProgressBar.html

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

This is very similar to #3423, if not a duplicate.
